executor/infoschema_reader_test.go

```diff
@@ -22,14 +22,17 @@ import (
 	"strings"
 	"time"
 
+	"context"
```

Comment:
Please optimize imports.
1. Remove the L24. 
2. Add an blank line after L25 (Put all 3rd dependencies into single group).

Reply:
Done!